### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by coldpigli**
Added a landing page
also a popular quizzes sectio
quiz-landing-page
landing-page-pop-quiz
n

### CODE REVIEW

**Review by SimranjitKaurMaan — APPROVED** [review]

File: styles.css [hunk 1]
```diff
@@ -15,16 +16,105 @@
...
+}
+
+.hero-cta>button{
+    width: 15rem;
+    border: none;
+    border-radius: 0%;
+    background-color: var(--pastel-blue);
+}
+
+.hero-cta>button:nth-child(2){
```
Comment (krishanu2911) [inline styles.css:60]: Just confirming that if it is for specific usage then it ok or else if not you can make a common class

**Review by krishanu2911 — COMMENTED** [review]
Everything Seems good to go Just Added a small confirmation else everything is clean and well written code

File: styles.css (hunk 1 above)
Comment (coldpigli) [reply styles.css:60]: It's for a specific usecase

Comment (krishanu2911) [reply styles.css:60]: then it's fine

**Review by krishanu2911 — APPROVED** [review]

**Pull request merged**